### PR TITLE
ZAM:

### PR DESCRIPTION
**Pull request by numismatika**
- comment about another subdomain with invalid cert

### CODE REVIEW

**Comment by J0WI** [comment]
Please note that this rule is `default_off`. Would you like to re-enable it?

**Comment by J0WI** [comment]
ping @numismatika

**Comment by numismatika** [comment]
@J0WI working on it

File: src/chrome/content/rules/Zam.xml [hunk 1]
```diff
@@ -1,30 +1,82 @@
...
+	<target host="loldata.services.zam.com" />
+	<target host="media.services.zam.com" />
+	<target host="media-staging.services.zam.com" />
+	<target host="parkplace-staging.services.zam.com" />
 
 	<target host="cdn.zamimg.com"/>
 	<target host="descss.zamimg.com"/>
 	<target host="desimg.zamimg.com"/>
 	<target host="desjs.zamimg.com"/>
-	<target host="lkcss.zamimg.com"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Zam.xml]: Can we keep a comment for it?

File: src/chrome/content/rules/Zam.xml [hunk 2]
```diff
@@ -1,30 +1,82 @@
...
 	<target host="lkimg.zamimg.com"/>
-	<target host="lkjs.zamimg.com"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Zam.xml]: Can we keep a comment for it?

File: src/chrome/content/rules/Zam.xml (hunk 1 above)
Comment (numismatika) [reply src/chrome/content/rules/Zam.xml]: both of these seem to no longer be in use on lolking and replaced by content on the .services subdomain

Comment (J0WI) [reply src/chrome/content/rules/Zam.xml]: But they are still accessible.

Comment (numismatika) [reply src/chrome/content/rules/Zam.xml]: mhmm i think i removed them for not having proper test urls and non 200 status codes or some sort of error like that, let me check that again

Comment (J0WI) [reply src/chrome/content/rules/Zam.xml]: You can just keep them as comments.

**Pull request merged**